src/views/registration-js/content.php

```diff
@@ -28,6 +28,12 @@
 	$provider     = $provider_obj->attendee_object;
 }
 
+if ( method_exists( $provider_obj, "get_cart_provider" ) ) {
```

Comment:
:no_entry_sign: String "get_cart_provider" does not require double quotes; use single quotes instead

<sub>hash: d82f83fdf9f29921dc688cc83404cd87</sub>

Reply:
```suggestion
if ( method_exists( $provider_obj, 'get_cart_provider' ) ) {
```